source/chapters/user_interface.rst

```diff
@@ -707,38 +707,123 @@ Channel Faders and Level Meters
   .. hint:: Some DJ's prefer to use the line faders over the crossfader for
             fading between tracks. Try it, you may like it.
 
-.. _interface-expansion-buttons:
+.. _interface-eq-gain:
 
-Section Expansion Buttons
+Equalizers and Gain Knobs
 -------------------------
 
-  .. figure:: ../_static/2.2/expansion-buttons-top.png
+.. figure:: ../_static/2.2/gain-knob.png
```

Comment:
I realize we need an image here that also shows the EQ Kill & QuickEffect Enable buttons.
Could you please add the image below?
![eq-gain-quickeffect-knobs](https://user-images.githubusercontent.com/5934199/98444806-13316980-2114-11eb-83f4-8fa3c0c8aaea.png)

Reply:
Hello @ronso0, from the changes I think you want me to change the image name from  ` .. figure:: ../_static/2.2/expansion-buttons-top.png` to
`.. figure:: ../_static/2.2/gain-knob.png` and also replace the image with the one above? I am just asking to clarify.
Thank you!

Reply:
This change has been effected.